js/src/widgets/metadataView.js

```diff
@@ -164,11 +163,7 @@
     var label = attr;
 
     label = label.replace(/^@/, '');
-    label = label.replace(/([A-Z])/g, ' $1');
     label = label.replace(/\s{2,}/g, ' ');
-    label = label.replace(/\w\S*/g, function(txt) {
```

Comment:
Can you actually leave these lines in and remove the relevant line in the CSS https://github.com/ProjectMirador/mirador/blob/2.1.3/css/mirador.css#L1802?  This does a better job at capitalizing than the CSS does (see `Creator(S)`):

![screen shot 2017-01-11 at 2 49 40 pm](https://cloud.githubusercontent.com/assets/2371632/21863938/6e2e3b8c-d80d-11e6-9b53-444959f1a64f.png)